main.cpp

```diff
@@ -376,6 +389,10 @@ struct GymChain
     Gym gym;
     Gym::Member member1;
     Gym::Member member2;
+    int gymCounter = 1;
+    int mChainLocations = 10;
+    int mFranchiseLocations = 17;
```

Comment:
don't use `m` in member variable names
If a variable is used in a function, and it is not declared in that function or part of the function parameter list, it is obvious that the variable is a member variable. 
- [ ] revise